README.md

```diff
@@ -162,6 +162,21 @@ Whether the system is connected to Insights; valid values are `present`
 Whether the system automatically updates the dynamic configuration. It is
 enabled by default.
 
+```yaml
+    rhc_insights:
+      ansible_host: null
```

Comment:
like the other examples that set string values, I'd specify an example of text/string